lib/hal/led.c

```diff
@@ -0,0 +1,32 @@
+/*
+Copyright and related rights waived via CC0
+https://creativecommons.org/publicdomain/zero/1.0/
+*/
+
+#include <hal/led.h>
+#include <xboxkrnl/xboxkrnl.h>
+#include <assert.h>
+
+#define SMC_REG_LEDMODE 0x07
+#define SMC_REG_LEDSEQ  0x08
+
+void XSetAutoLED(void) {
+  HalWriteSMBusValue(0x20, SMC_REG_LEDMODE, FALSE, 0x00);
+}
+
+void XSetManualLED(unsigned int t1, unsigned int t2, unsigned int t3, unsigned int t4) {
+  HalWriteSMBusValue(0x20, SMC_REG_LEDMODE, FALSE, 0x01);
```

Comment:
Please add constant definitions for LED mode and use them here. ([See here](https://github.com/rkalz/xbmc4xbox/blob/50f38fb2fce74f2d7679ffc14f3d5002b9c65b6a/xbmc/xbox/XKUtils.h#L134))